src/functions/template-tags/general.php

```diff
@@ -288,6 +288,33 @@ function tribe_event_is_multiday( $postId = null ) {
 	}
 
 	/**
+	 * Detect if an Event is Past
+	 *
+	 * Returns true if the current time is past the event end time
+	 *
+	 * @param null $event
+	 *
+	 * @return bool
+	 */
+	function tribe_is_past_event( $event = null ) {
+
+		if ( is_null( $event ) ) {
+			global $post;
+			$event = $post;
+		}
+		// Check if event has passed
+		$gmt_offset = ( get_option( 'gmt_offset' ) >= '0' ) ? ' +' . get_option( 'gmt_offset' ) : ' ' . get_option( 'gmt_offset' );
+		$gmt_offset = str_replace( array( '.25', '.5', '.75' ), array( ':15', ':30', ':45' ), $gmt_offset );
+
+		if ( ! tribe_is_showing_all() && strtotime( tribe_get_end_date( $event, false, 'Y-m-d G:i' ) . $gmt_offset ) <= time() ) {
```

Comment:
... If we were to drop it from this lcoation, we could do the following back in `set_notices()`:

``` php
if ( ! tribe_is_showing_all() && tribe_is_past_event() ) {
    ...
}
```